Traceback (most recent call last):
  File "/n/home05/alanhsu/workspace/research/continual_learning/main_batch.py", line 239, in <module>
    main(config)
  File "/n/home05/alanhsu/workspace/research/continual_learning/main_batch.py", line 68, in main
    os.mkdir(model_save_dir)
FileNotFoundError: [Errno 2] No such file or directory: 'models/cifar10_split'
  0%|          | 0/170498071 [00:00<?, ?it/s]  0%|          | 229376/170498071 [00:00<01:17, 2186126.36it/s]  2%|▏         | 3178496/170498071 [00:00<00:09, 17879390.93it/s]  7%|▋         | 11141120/170498071 [00:00<00:03, 45652920.18it/s] 11%|█▏        | 19464192/170498071 [00:00<00:02, 60272827.64it/s] 16%|█▋        | 27787264/170498071 [00:00<00:02, 68469081.84it/s] 21%|██        | 36012032/170498071 [00:00<00:01, 73087409.31it/s] 26%|██▌       | 44466176/170498071 [00:00<00:01, 76643814.86it/s] 31%|███       | 52920320/170498071 [00:00<00:01, 79023182.07it/s] 36%|███▌      | 61177856/170498071 [00:00<00:01, 80068477.33it/s] 41%|████      | 69664768/170498071 [00:01<00:01, 81312685.82it/s] 46%|████▌     | 78086144/170498071 [00:01<00:01, 82003528.99it/s] 51%|█████     | 86507520/170498071 [00:01<00:01, 82414632.16it/s] 56%|█████▌    | 94765056/170498071 [00:01<00:01, 74717192.23it/s] 60%|██████    | 102400000/170498071 [00:01<00:00, 69218776.51it/s] 64%|██████▍   | 109477888/170498071 [00:01<00:00, 66089621.95it/s] 68%|██████▊   | 116228096/170498071 [00:01<00:00, 63823608.05it/s] 72%|███████▏  | 122716160/170498071 [00:01<00:00, 62407682.73it/s] 76%|███████▌  | 129007616/170498071 [00:01<00:00, 61306337.52it/s] 79%|███████▉  | 135168000/170498071 [00:02<00:00, 60556280.90it/s] 83%|████████▎ | 141262848/170498071 [00:02<00:00, 59926542.76it/s] 86%|████████▋ | 147292160/170498071 [00:02<00:00, 59604679.43it/s] 90%|████████▉ | 153288704/170498071 [00:02<00:00, 59250452.19it/s] 93%|█████████▎| 159219712/170498071 [00:02<00:00, 58947271.64it/s] 97%|█████████▋| 165117952/170498071 [00:02<00:00, 58866967.34it/s]100%|██████████| 170498071/170498071 [00:02<00:00, 64637267.30it/s]
Traceback (most recent call last):
  File "/n/home05/alanhsu/workspace/research/continual_learning/main_batch.py", line 239, in <module>
    main(config)
  File "/n/home05/alanhsu/workspace/research/continual_learning/main_batch.py", line 117, in main
    post_train_model = torch.load(post_train_model_load_path)
  File "/n/home05/alanhsu/.conda/envs/continualLearning/lib/python3.9/site-packages/torch/serialization.py", line 986, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/n/home05/alanhsu/.conda/envs/continualLearning/lib/python3.9/site-packages/torch/serialization.py", line 435, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/n/home05/alanhsu/.conda/envs/continualLearning/lib/python3.9/site-packages/torch/serialization.py", line 416, in __init__
    super().__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: 'models/cifar10_split/ideal_model/task_0/model.pt'
